### PR TITLE
net-proxy/macchanger: fix compilation with musl libc

### PR DESCRIPTION
**Pull request by doughdemon**
@robbat2

### CODE REVIEW

File: net-analyzer/macchanger/macchanger-1.7.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+OUI_DATE="20091029" # Generated with tools/IEEE_OUI.py in the source
+OUI_FILE="OUI.list-${OUI_DATE}"
+HOMEPAGE="https://github.com/alobbs/macchanger"
+SRC_URI="https://github.com/alobbs/macchanger/releases/download/${PV}/${P}.tar.gz"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~arm ~ppc ~sparc ~x86"
+SLOT="0"
+
+PATCHES=( "${FILESDIR}"/${P}-fix-caddr_t.patch )
+DOCS=( AUTHORS ChangeLog NEWS README )
```
Comment (SoapGentoo) [inline net-analyzer/macchanger/macchanger-1.7.0-r1.ebuild]: don't need these, part of the default set

File: net-analyzer/macchanger/macchanger-1.7.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+src_configure() {
+	# Shared data is installed below /lib, see Bug #57046
+	econf \
+		--bindir="${EPREFIX%/}/sbin" \
+		--datadir="${EPREFIX%/}/lib"
```
Comment (SoapGentoo) [inline net-analyzer/macchanger/macchanger-1.7.0-r1.ebuild]: `EPREFIX` has never been slashed-terminated, `%/` are pointless

**Comment by doughdemon** [comment]
> +DOCS=( AUTHORS ChangeLog NEWS README )
> 
> don't need these, part of the default set

I tried first without declaring `DOCS` but then NEWS did not get
installed.

**Comment by doughdemon** [comment]
`NEWS` was not installed because it was empty...
Now all your requested changes are implemented.

**Comment by SoapGentoo** [comment]
@doughdemon could you please rebase to get rid of the CI error.

**Comment by robbat2** [comment]
merged in 934b2a3612015642792d69007c6243f6a5f123c6

**Pull request closed without merging**